.github/workflows/pr.yml

```diff
@@ -0,0 +1,17 @@
+name: PR
+on: pull_request
+jobs:
+  pr:
+    runs-on: ubuntu-18.04
```

Comment:
```suggestion
    runs-on: ubuntu-20.04
```
Let's stick to something a bit more recent?

Reply:
From https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
![image](https://user-images.githubusercontent.com/506752/99302717-a0af4f00-28b4-11eb-9db0-63917feb6860.png)

It probably wouldn't harm, but..

Reply:
discussed via slack, we'll stay with 18.04 for now